src/Framework/Traits.cs

```diff
@@ -35,6 +35,11 @@ public Traits()
 
         public EscapeHatches EscapeHatches { get; }
 
+        /// <summary>
+        /// https://github.com/dotnet/msbuild/issues/3897
+        /// </summary>
+        public readonly bool EnableHardcodedCultureNames = Environment.GetEnvironmentVariable("MSBUILDENABLEHARDCODEDCULTURENAMES") == "1";
```

Comment:
I thought we were thinking opt out? But I probably misremembered.

Reply:
Yeah this will turn into a change wave instead of a trait